File: Basic/code/problem_6.c
/*
Write a program to Input base and height and calculate area of triangle.
*/

// Solved by napalion

#include<stdio.h>
int main(){
    int height,base;
    printf("Enter the base: \n");
    scanf("%d",&base);
    printf("Enter the height: \n");
    scanf("%d",&height);
    printf("The area of triangle of base %d and height %d is %d\n",base,height,(height*base)/2);
    return 0;
}

File: Basic/Java/Problem_6.java
import java.util.*;
public class Main
{
  public static void main (String[]args)
  {
     float base, height;
     Scanner input = new Scanner (System.in);

      System.out.println ("Enter Base value  : ");
      base = input.nextFloat ();
      System.out.println ("Enter height value  : ");
      height = input.nextFloat ();
      
      System.out.print ("Area Of The Triangle : ");
      System.out.println (getAreaOfTriangle (base, height));
  }
  private static double getAreaOfTriangle (float base, float height)
  {
    double area;
    area = (0.5 * base * height);
    return area;

  }
}
